pom.xml

```diff
@@ -158,8 +158,7 @@ under the License.
 		<beam.version>2.43.0</beam.version>
 		<protoc.version>3.21.7</protoc.version>
 		<okhttp.version>3.14.9</okhttp.version>
-		<!-- keep net.java.dev.jna:jna version in sync -->
```

Comment:
no more need for such sync after this was merged
https://github.com/apache/flink/pull/22474